### PR TITLE
Fix "Password Reset by PIN" page

### PR DESCRIPTION
**Pull request by grafixeyehero**

### CODE REVIEW

File: src/controllers/forgotpassword.js [hunk 1]
```diff
@@ -12,9 +12,12 @@ define([], function() {
         });
         if ("PinCode" == result.Action) {
             var msg = Globalize.translate("MessageForgotPasswordFileCreated");
-            return msg += "<br/>", msg += "<br/>", msg += "<a href=\"forgotpasswordpin.html\">Enter PIN here to finish Password Reset</a>" ,msg += "<br/>",msg += result.PinFile, msg += "<br/>", void Dashboard.alert({
+            return msg += "<br/>", msg += "<br/>", msg += "Enter PIN here to finish Password Reset<br/>" ,msg += "<br/>",msg += result.PinFile, msg += "<br/>", void Dashboard.alert({
                 message: msg,
                 title: Globalize.translate("HeaderForgotPassword")
+				callback: function() {
```
Comment (cvium) [inline src/controllers/forgotpassword.js]: Weird indentation. Do you use tabs?

File: src/controllers/forgotpassword.js [hunk 2]
```diff
@@ -12,9 +12,12 @@ define([], function() {
         });
         if ("PinCode" == result.Action) {
             var msg = Globalize.translate("MessageForgotPasswordFileCreated");
-            return msg += "<br/>", msg += "<br/>", msg += "<a href=\"forgotpasswordpin.html\">Enter PIN here to finish Password Reset</a>" ,msg += "<br/>",msg += result.PinFile, msg += "<br/>", void Dashboard.alert({
+            return msg += "<br/>", msg += "<br/>", msg += "Enter PIN here to finish Password Reset<br/>" ,msg += "<br/>",msg += result.PinFile, msg += "<br/>", void Dashboard.alert({
                 message: msg,
                 title: Globalize.translate("HeaderForgotPassword")
```
Comment (dkanada) [inline src/controllers/forgotpassword.js]:
```suggestion
                title: Globalize.translate("HeaderForgotPassword"),
```

Comment (dkanada) [inline src/controllers/forgotpassword.js]:
```suggestion
                 title: Globalize.translate("HeaderForgotPassword"),
```

Comment (grafixeyehero) [reply src/controllers/forgotpassword.js]: @dkanada this is done thanks for finding

**Pull request merged**